Traceback (most recent call last):
  File "main.py", line 5, in <module>
    a = int(a)
NameError: name 'a' is not defined
